src/calculateRentalCost.js

```diff
@@ -3,8 +3,23 @@
  *
  * @return {number}
  */
+
+const basePrice = 40;
+const threeDays = 3;
+const threeDaysDiscount = 20;
+const sevenDays = 7;
+const sevenDaysDiscount = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  if (days < threeDays) {
+    return basePrice * days;
+  }
+
+  if (days < sevenDays) {
+    return basePrice * days - threeDaysDiscount;
```

Comment:
The condition `days < sevenDays` is correct, but it's not immediately clear why the `threeDaysDiscount` is being applied here. Adding a comment to explain that this branch handles the case for rentals of 3 to 6 days would improve code readability.